### PR TITLE
Updated README.md, CONTRIBTING.md, README logo and README Screenshot

### PR DESCRIPTION
**Pull request by SkogiB**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,28 +16,26 @@ which details these points and more, and is a recommended read.
...
+Redot project which reproduces the issue, with no unnecessary files included.
 Be sure to not include the `.godot` folder in the archive to save space.
 
-Make sure that the bug you are experiencing is reproducible in the latest Godot
-releases. You can find an overview of all Godot releases [on the website](https://godotengine.org/download/archive/)
-to confirm whether your current version is the latest one. It's worth testing
-against both the latest stable release and the latest dev snapshot for the next
-Godot release.
+Make sure that the bug you are experiencing is reproducible in the latest Redot
+releases. It's worth testing against both the latest stable release and the latest dev snapshot for the next
```
Comment (Starkium) [inline CONTRIBUTING.md]: Need reformatting

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,18 +53,7 @@ Similar rules can be applied when contributing bug fixes - it's always best to
...
-
-In addition to the following tips, also take a look at the
-[Engine development guide](https://docs.godotengine.org/en/latest/contributing/development/index.html)
-for an introduction to developing on Godot.
-
-The [Contributing docs](https://docs.godotengine.org/en/latest/contributing/how_to_contribute.html)
-also have important information on the [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html)
-(with a helpful guide for Git usage), and our [Code style guidelines](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html)
-which all contributions need to follow.
+Visit the [Redot Discord](https://discord.gg/redot) for information and guides related to contribution and pull requests.
```
Comment (Starkium) [inline CONTRIBUTING.md:55]: we don't have this stuff

**Comment by Starkium** [comment]
Just changing links in place isn't going to work. We need to think about what we're going to say since we don't have any documentation or formal protocols up. We might just need to revisit this one we at least have some sort of landing page up.

File: CONTRIBUTING.md (hunk 1 above)
Comment (SkogiB) [reply CONTRIBUTING.md]: Need more detailed description than that. It looks good on my local repo and its the same text as the Godot original, just with a sentence removed

File: CONTRIBUTING.md (hunk 2 above)
Comment (SkogiB) [reply CONTRIBUTING.md:55]: Ah, missed that one, on it

**Comment by SkogiB** [comment]
> Just changing links in place isn't going to work. We need to think about what we're going to say since we don't have any documentation or formal protocols up. We might just need to revisit this once we at least have some sort of landing page up.

I missed one link (its been to address a change request though), but the rest were scrubbed or changed. I left a few links to Godot's site in for things like build instructions, since that seems to critical to scrub, but other than that it should be good

**Comment by SkogiB** [comment]
Something is messed up and I can't push the final updates to the files to my repo, hang tight

**Comment by SkogiB** [comment]
idk if force push bad, things were busted, these are the files with all corrections made

**Comment by SkogiB** [comment]
Discord Handle .skogi

**Pull request merged**